proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -96,6 +96,38 @@ message SignDocJSON {
   bytes sign_doc_sha256_hash = 5;
 }
 
+// SignDocDirectAux is the type used for generating sign bytes for
+// SIGN_MODE_DIRECT_AUX.
+message SignDocDirectAux {
+  // body_bytes is protobuf serialization of a TxBody that matches the
+  // representation in TxRaw.
+  bytes body_bytes = 1;
+
+  // public_key is the public key of the signing account.
+  google.protobuf.Any public_key = 2;
+
+  // chain_id is the identifier of the chain this transaction targets.
+  // It prevents signed transactions from being used on another chain by an
+  // attacker.
+  string chain_id = 3;
+
+  // account_number is the account number of the account in state.
+  uint64 account_number = 4;
+
+  // sequence is the sequence number of the signing account.
+  uint64 sequence = 5;
+
+  // Fee is the fee and gas limit for the transaction. The first signer is the
+  // primary signer and the one which pays the fee. The fee can be calculated
+  // based on the cost of evaluating the body and doing signature verification
+  // of the signers. This can be estimated via simulation.
+  Fee fee = 6;
```

Comment:
@aaronc I added Fee on top of your version of SignDocAux. Now everything that was in the signer's AuthInfo is in SignDoc too (for 0 malleability surface)

Reply:
`SignDocAux` can't contain `Fee`. That defeats the whole purpose of aux - if they knew the fee, they wouldn't be using SIGN_MODE_AUX. Basically fee payers should always use SIGN_MODE_DIRECT and non-fee payers can use SIGN_MODE_AUX and then they don't need to sign over the fee or any of the rest of AuthInfo

Reply:
Got it, thanks. I reverted `Fee`